src/containers/Header/NetworkPicker/NetworkPicker.tsx

```diff
@@ -116,12 +140,12 @@ export const NetworkPicker = () => {
       <>
         {networks.map(({ network, title, url = '' }) => {
           if (
-            network === currentMode ||
+            (currentMode !== 'custom' && network === currentMode) ||
```

Comment:
Can you add a comment explaining what is going on here.  I forgot what the reasoning is.